pageserver/src/lib.rs

```diff
@@ -53,6 +53,7 @@ pub use crate::metrics::preinitialize_metrics;
 
 #[tracing::instrument(skip_all, fields(%exit_code))]
 pub async fn shutdown_pageserver(deletion_queue: Option<DeletionQueue>, exit_code: i32) {
+    tracing::info!("shutdown: starting shutdown...");
```

Comment:
does it make sense to prefix these as there is already a `shutdown_pageserver` span?

Reply:
Perhaps you are adding the prefix to mask some other problem?

Reply:
These are gone now.